packages/inter-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -336,12 +333,26 @@ const start = async zcf => {
     );
   };
 
+  const LiquidateProposalShape = M.splitRecord({
```

Comment:
I believe the liquidation files are going away too so won't review this.

Reply:
In light of that, I reverted this file. Resolving.